ui/src/components/chat/compose.rs

```diff
@@ -343,25 +343,34 @@ fn get_chatbar(cx: Scope<ComposeProps>) -> Element {
             )),
             with_replying_to: data.map(|data| {
                 let active_chat = data.active_chat.clone();
+                let participants = build_participants(&data.active_chat.participants.clone());
+
                 cx.render(rsx!(
-                    active_chat.clone().replying_to.map(|msg| rsx!(
+                    active_chat.clone().replying_to.map(|msg|
+                    {
+                    let our_did = state.read().account.identity.did_key();
+                    let their_did = msg.sender();
+                    let (platform, status) = if our_did == their_did {
+                        (participants[1].platform, participants[1].status)
```

Comment:
Would be careful when using an index like that. Would recommend using something like `participants.get(1)` and check to see if that is `Some` or `None`. Just mostly to prevent any possible panic due to it being out of bounds for whatever reason (though we can assume/expect it to be more than 1 participant here with any panics being considered a bug, but one can sometimes gracefully handle it without a panic)